### DISCUSSION TITLE
Can OME output SRT?

### DISCUSSION THREAD

**Discussion by tonywingmah** [OvenMediaLabs/OvenMediaEngine#1227] [Help] [answered]
We are using Haivision PlayPro phone app for our production. Can I send SRT from our encoders to OME and have OME act as the listener and have a mobile phone as a SRT caller to OME?

**Comment by getroot** [comment, marked as answer]
The title and content confuse me.
Do you mean SRT restream? Or do you mean SRT Ingest?
See #1228 for SRT restream. SRT Ingest is supported. Please refer to the manual.
